### PR TITLE
corrections to formal spec

### PR DESCRIPTION
**Pull request by polinavino**
corrections, with notes at
https://github.com/input-output-hk/cardano-ledger-specs/pull/1330

### CODE REVIEW

**Comment by polinavino** [comment]
@jmchapman @nfrisby let me know if there is anything missing/confusing/incorrect

File: shelley/chain-and-ledger/formal-spec/address.tex [hunk 1]
```diff
@@ -3,10 +3,18 @@ \section{Addresses}
...
+They all involve a credential, which is either a key or a multi-signature script.
 There are four types of UTxO addresses:
 \begin{itemize}
 \item Base addresses, $\AddrB$, containing the hash of a payment credential and
-  the hash of a staking credential,
+  the hash of a staking credential. Note that the payment credential hash is the
+  hash of the key (or script) which has contol of the funds at this address,
+  i.e. is able to witness spending
+  them. The staking credential is the credential which accumulates
+  the rewards associated with the funds at the payment part of the address.
```
Comment (JaredCorduan) [inline shelley/chain-and-ledger/formal-spec/address.tex]: Is it maybe confusing here to conflate the stake key and the reward address? Maybe we could say that the staking credential controls the delegation decision?

File: shelley/chain-and-ledger/formal-spec/address.tex [hunk 2]
```diff
@@ -46,7 +53,16 @@ \section{Addresses}
 rewards from the proof of stake leader election. Appendix A
 of~\cite{delegation_design} explains this design choice.  The mechanism for
 transferring rewards from these accounts will be explained in
-Section~\ref{sec:utxo} and follows~\cite{chimeric}.
+Section~\ref{sec:utxo} and follows the approach outlined in the
+document~\cite{chimeric}.
+
+Note that, even though in the Cardano system, most of the
+accounting is UTxO-style, the reward addresses are a special case. They do not
+fall prey to many of the pitfalls of account-based accounting because of their restricted
```
Comment (JaredCorduan) [inline shelley/chain-and-ledger/formal-spec/address.tex]: I'm not sure I like the idea of coming down so hard on this style without justification. Maybe just say that this style is only used here?

File: shelley/chain-and-ledger/formal-spec/address.tex (hunk 1 above)
Comment (polinavino) [reply shelley/chain-and-ledger/formal-spec/address.tex]: Sounds good

**Pull request merged**